StitchCore/Sources/StitchCore/Auth/Providers/UserAPIKey/CoreUserAPIKeyAuthProviderClient.swift

```diff
@@ -113,7 +113,7 @@ open class CoreAuthenticatedUserAPIKeyClient: CoreAuthProviderClient<StitchAuthR
         return try decode(fromResponse: self.requestClient.doAuthenticatedRequest(
             StitchAuthRequestBuilderImpl {
             $0.method = .get
-            $0.path = self.routes.apiKeyRoute(forKeyId: id.hexString)
+            $0.path = self.routes.apiKeyRoute(forKeyId: id.description)
```

Comment:
[minor] I know this is on the driver and not you, but I don't really like this. I feel like `hexString` is a better description of the value than `description`. Can we file a ticket with the Swift driver to expose a `hexString` property? Java has `toHexString()` so I don't see a reason not to.

Reply:
I'm actually a bit torn on this one. I think it should probably be both– `description` is the string representation of an ObjectId, which is generally adhered to in Swift. There are no other properties of an ObjectId that matter. I can bring it up with SWIFT though.

Reply:
I definitely think it should be `hexString`. `description` could have very well been `ObjectIdHex('blah')`